snippets/csharp/concepts/methods/params75.cs

```diff
@@ -0,0 +1,44 @@
+//<Snippet75>
+using System;
+using System.Linq;
+
+class Example
+{
+    static void Main()
+    {
+        string fromArray = GetVowels(
+            new[] { "apple", "banana", "pear" });
+
+        string fromParameters = GetVowels(
+            "apple", "banana", "pear");
+
+        string fromSingleValue = GetVowels(
```

Comment:
I'm not sure what you're driving for on this part of the sample. 

Can I suggest a draft PR in the dotnet/docs repo on the "methods" topic so I can get more understanding of your intent? It would help me review this.

Reply:
Sure, I'll do that.

Reply:
See https://github.com/dotnet/docs/pull/12763